modules/posts/includes/class-post-view-base.php

```diff
@@ -23,4 +46,721 @@ function post() {
+		));
+
+		$this->get_terms_list_links_html( $args );
+
+	}
+
+
+	/**
+	 * @param array $args
+	 */
+	function the_post_tag_list_links_html( $args = array() ) {
+
+		echo wp_kses_post( $this->get_post_tag_list_links_html( $args ) );
+
+	}
+	/**
+	 * @param array $args
+	 *
+	 * @return bool|string
+	 */
+	function get_post_tag_list_links_html( $args = array() ) {
+
+		$args = wp_parse_args( $args, array(
+			'html_template' => __( 'Tagged %1$s', 'wplib' ),
+			'before'        => '<span class="tags-links{{class}}">',
+		));
+
+		$this->get_terms_list_links_html( $args );
+
+	}
+
+	/**
+	 * @param array $args
+	 *
+	 * @return bool|string
+	 */
+	function get_terms_list_links_html( $args = array() ) {
+
+		$args = wp_parse_args( $args, array(
+			/*
+			 * translators: used between list items, there is a space after the comma
+			 */
+			'separator'     => __( ', ', 'wplib' ),
+			'parents'       => '',
+			'html_template' => __( 'Posted in %1$s', 'wplib' ),
+			'before'        => '<span class="term-links{{class}}">',
+			'after'         => '</span>',
+			'class'         => false,
+			'taxonomy'      => false,
+
+		) );
+
+		$args['class'] = $args['class'] ? " {$args[ 'class' ]}" : '';
+
+		$args['before'] = str_replace( '{{class}}', $args['class'], $args['before'] );
+
+		switch( $args[ 'taxonomy' ] ) {
+			case WPLib_Category::TAXONOMY:
+
+				$list = get_the_category_list(
+					esc_html( $args['separator'] ),
+					$args['parents'],
+					$this->ID()
+				);
+				break;
+
+			case WPLib_Post_Tag::TAXONOMY:
+
+				$list = get_the_tag_list(
+					'', // before
+					esc_html( $args['separator'] ),
+					'', // after
+					$this->ID()
+				);
+				break;
+
+			default:
+
+				$list = get_the_term_list(
+					$this->ID(),
+					$args[ 'taxonomy' ],
+					'', // before
+					esc_html( $args['separator'] ),
+					'' // after
+				);
+				break;
+
+		}
+
+		if ( ! $list ) {
+
+			$html = false;
+
+		} else {
+
+			$args['html_template'] = esc_html( $args['html_template'] );
+
+			$html = sprintf( "{$args[ 'before' ]}{$args[ 'html_template' ]}{$args[ 'after' ]}", $list );
+
+		}
+
+		return $html;
+
+	}
+
+	/**
+	 * @param array $args
+	 *
+	 * @return mixed|null
+	 */
+	function get_comments_html( $args = array() ) {
+
+		ob_start();
+
+		$this->the_comments_html( $args );
+
+		return ob_get_clean();
+
+	}
+
+	/**
+	 * @param array $args
+	 *
+	 * @return mixed|null
+	 */
+	function the_comments_html( $args = array() ) {
+
+		if ( ! $this->has_post() )  {
+
+			$comments_html = null;
+
+		} else {
+
+			global $post;
+
+			$args = wp_parse_args( $args, array(
+				'template_file'  => '/comments.php',
+				'group_by_type'  => false,
+			) );
+
+			$save_post = $post;
+
+			$post = $this->model()->post();
+
+			comments_template( $args[ 'template_file' ], $args[ 'group_by_type' ] );
+
+			$post = $save_post;
+
+		}
+
+		return $comments_html;
+
+	}
+
+	/**
+	 *
+	 */
+	function the_content_html() {
+
+		echo wp_kses_post( $this->get_content_html() );
```

Comment:
No.  If the method exists it will be called.  Auto-escaping only happens for virtual methods.

Do you think it should?  _(though I do not know that would be possible.)_